public_suffix_list.dat

```diff
@@ -12828,6 +12828,8 @@ mcpe.me
 mcdir.me
```

Comment:
Please add a non-personal e-mail address above which stays reachable if you should leave the company.